tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -641,18 +690,29 @@ class LineChart {
       yDomain = this._defaultYRange;
     } else {
       // Generate a reasonable range.
-      const accessor = this.getAccessor();
-      let datasetToValues: (d: Plottable.Dataset) => number[] = (d) => {
-        return d.data().map((x) => accessor(x, -1, d));
+      const accessors = this.getAccessorsForComputingYRange();
+      let datasetToValues: (d: Plottable.Dataset) => number[][] = (d) => {
+        return accessors.map(accessor => d.data().map(x => accessor(x, -1, d)));
       };
-      let vals = _.flatten(this.datasets.map(datasetToValues));
-      vals = vals.filter((x) => x === x && x !== Infinity && x !== -Infinity);
+      let vals = _.flattenDeep<number>(this.datasets.map(datasetToValues));
```

Comment:
This is monadic, and can just be
```ts
const vals = _.flatMap(d, d => _.flatMap(accessors, accessor =>
  d.data().map(accessor(x, -1, d)).filter(isFinite)));
```
(or move `.filter(isFinite)` to the outside, if you prefer).

This reads much more easily to me; what do you think?

Reply:
Nice! `_.flatMap` seems apt here. Done.

Reply:
Ah, I switched back to using `_.flattenDeep` because our current typings for lodash lack  `_.flatMap`.